src/Components/ListArticlesByTopic.jsx

```diff
@@ -0,0 +1,24 @@
+import { fetchArticlesByTopic } from "../api";
+import { useEffect } from "react";
+import { useState } from "react";
+import { Article } from "./Article";
+
+export const ListArticlesByTopic = ({ topic }) => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticlesByTopic(topic).then((articleData) => {
+      setArticles(articleData);
+    });
+  }, []);
+
+  return (
+    <div>
+      <th id="articles_th">
+        {articles.map((articles) => {
+          return <Article article={articles} />;
```

Comment:
the console is also flagging the tr cannot be a child of th. I'd recommend looking up these tags on w3 schools to check how they need to be laid out.